src/videoframeprocessor/DefaultVideoFrameProcessorPipeline.ts

```diff
@@ -149,7 +149,13 @@ export default class DefaultVideoFrameProcessorPipeline implements VideoFramePro
     this.sourceBuffers.push(canvasBuffer);
 
     this.videoInput.load();
-    await this.videoInput.play();
+    this.videoInput.play()
+      .then(() => {
+        this.logger.info('video element started playing.');
+      })
+      .catch(() => {
+        this.logger.warn('Video element play() overrided by another load().');
```

Comment:
Is this a overridden action or an interrupted action caused by another load?

Reply:
Another call to load() is being called before the previous play() could finish - we should just let the new load() command take over